automation/README.md

```diff
@@ -43,9 +43,11 @@ FIAB=true ./config/docker-rsync-local-ui.sh
 
 #### From IntelliJ
 
-In IntelliJ, go to `Run` > `Edit Configurations...`, select `ScalaTest` under `Defaults`, and make sure there is a `Build` task configured to run before launch.
+In IntelliJ, go to `Run` > `Edit Configurations...`, select `ScalaTest` under `Templates`, and make sure there is a `Build` task configured to run before launch.
```

Comment:
We were noticing the automation readme had some issues while trying to set it up, so decided to yak-shave and update it here.